sdk/communication/communication-call-automation/src/models/options.ts

```diff
@@ -6,32 +6,27 @@ import { OperationOptions } from "@azure/core-client";
 import {
   MediaStreamingConfiguration,
   CallRejectReason,
-  RecognizeInputType,
   FileSource,
   DtmfTone,
   RecordingContent,
   RecordingChannel,
   RecordingFormat,
-  RecordingStorage,
   CallLocator,
 } from "./models";
 
 /** Options to configure the recognize operation. */
 export interface CallMediaRecognizeOptions extends OperationOptions {
-  recognizeInputType: RecognizeInputType;
   playPrompt?: FileSource;
```

Comment:
should this be PlaySource? it is weird this only accepts this specific source of play

Reply:
@ikumarapeli-msft yeah this should be playsource

Reply:
to be done in separate PR